packages/client/mutations/handlers/handleUpdateSpotlightResults.ts

```diff
@@ -30,11 +30,11 @@ const handleUpdateSpotlightResults = (
   const reflectionGroupId = reflectionGroup.getValue('id')
   const groupsIds = similarReflectionGroups
     .map((group) => group.getValue('id'))
-    .concat(reflectionGroupId)
+    .concat(spotlightGroupId)
   const isInSpotlight = groupsIds.includes(reflectionGroupId)
   const wasInSpotlight = groupsIds.includes(oldReflectionGroupId)
   // added to another group. Remove old reflection group
-  if (isOldGroupEmpty) {
+  if (isOldGroupEmpty && wasInSpotlight) {
```

Comment:
`wasInSpotlight` isn't strictly necessary but it prevents the function from firing if groups in the kanban are ungrouped